test/e2e/sched/sched.go

```diff
@@ -21,61 +21,74 @@ import (
 	"fmt"
 	"time"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog/v2"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	nropv1alpha1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1alpha1"
-	"github.com/openshift-kni/numaresources-operator/pkg/status"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
 	e2eclient "github.com/openshift-kni/numaresources-operator/test/utils/clients"
 	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
 )
 
-var _ = Describe("[Scheduler] install", func() {
-	var initialized bool
+// TODO change to an image from openshift-kni registry
+const newTestImage = "quay.io/titzhak/kube-scheduler:test"
```

Comment:
yes, we need to have this fixed. We probably need a test image (created from the NROP sources by CI)

Reply:
If the test image is temporary, i.e. created during the CI run and deleted once the run is done, we'll have a problem with running the tests locally.

Maybe we can create an image and store it in a local registry?

Reply:
I think we just need to have a separate image with RTE only (not the default one with NROP+RTE) that we can use freely for this kind of test. This image should be built by CI, uploaded on quay and kept here.

Reply:
>This image should be built by CI, uploaded on quay and kept here.

AFAIK we don't have a procedure for that

Reply:
yep, we should set it up as prerequisite for this PR.
Alternatively, we need to get rid of the tests requiring the separate image from this PR, and postpone until we have CI set.

Reply:
I prefer the former because I need that for additional tests I'm writing.

Do we have an example for something similar in other projects/repo?

Reply:
we "just" need to build a image using a Dockerfile we already have (may require some updates/fixes): https://github.com/openshift-kni/numaresources-operator/blob/main/Dockerfile.openshift.rte
The PAO flow builds quite a lot of images in CI, could be a good starting point

Reply:
BTW I just noticed, we don't need RTE image here, but scheduler-plugins one:
https://github.com/openshift/release/pull/24949

Reply:
right, I misread the imagespec

Reply:
Now that we have the new tag, we should trigger it somehow